test/unit/foreman/renderer/snapshots/ProvisioningTemplate/provision/Kickstart default.snap.txt

```diff
@@ -10,7 +10,8 @@ keyboard us
 skipx
 
 
-network --bootproto dhcp --hostname snapshothost --device=00-f0-54-1a-7e-e0
+network --bootproto dhcp --hostname snapshothost 
```

Comment:
This seems to be an outcome of the vlan patch. is this expected @lzap ?

Reply:
Good catch, not what I wanted. Created a fix for that: https://github.com/theforeman/community-templates/pull/648